Backpack/CardButton/README.md

```diff
@@ -7,7 +7,7 @@
 ## Default
 | Day | Night |
 | --- | --- |
-| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone%208-cardbutton___all_lm.png" alt="" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone%208-cardbutton___all_dm.png" alt="" width="375" /> |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone-cardbutton___all_lm.png" alt="" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone-cardbutton___all_dm.png" alt="" width="375" /> |
```

Comment:
Can we also rename this to cardButton (or double check the naming of the screenshots?)
We probably want to set a 'standard' in our CONTRIBUTING.md